src/pandroid/app/src/main/java/com/panda3ds/pandroid/input/InputMap.java

```diff
@@ -0,0 +1,42 @@
+package com.panda3ds.pandroid.input;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+
+import com.panda3ds.pandroid.app.PandroidApplication;
+import com.panda3ds.pandroid.utils.Constants;
+
+public class InputMap {
+
+    private static SharedPreferences data;
+    private static final String KEY_DEAD_ZONE = "deadZone";
```

Comment:
Shouldn't this also be in Constants? Also what is this deadZone variable for?

Reply:
this is used only this class, this is a key for save a float in SharedPreferences, Constants is for global variables